src/components/NcActions/NcActions.vue

```diff
@@ -708,7 +708,8 @@ export default {
 		 * @return {boolean}
 		 */
 		isValidSingleAction(action) {
-			return ['NcActionButton', 'NcActionLink', 'NcActionRouter'].includes(action?.componentOptions?.tag)
+			const componentName = action?.componentOptions?.Ctor?.extendOptions?.name
```

Comment:
Maybe keep the tag as a fallback, just in case?
```suggestion
			const componentName = action?.componentOptions?.Ctor?.extendOptions?.name || action?.componentOptions?.tag
```